stargz/stargz.go

```diff
@@ -175,6 +177,9 @@ type TOCEntry struct {
 	ChunkSize   int64 `json:"chunkSize,omitempty"`
 
 	children map[string]*TOCEntry
+
+	// Nlink is the number of entry names pointing to this entry.
+	Nlink int
```

Comment:
Put this above the unexported children field.

I'd probably put it between DevMinor and ChunkOffset.

Let's name it NumLink too. And document what zero means. Does zero mean one?

Reply:
Thank you for your review. I fixed it and now NumLink=0 means one name references the entry.